Make webp optional dependenct

I'm not using any webp
But image-webpack-loader cannot be installed w/o webp support.
I have a problem with installing image-webpack-loader in CI 

<details><summary>log</summary>

```
Running with gitlab-runner 11.10.1 (1f513601)
  on Docker Sexy Runner 3mSBpTBe
Using Docker executor with image node:11.15.0-alpine ...
Pulling docker image node:11.15.0-alpine ...

$ npm i
...
> cwebp-bin@5.0.0 postinstall /builds/backend/felix/front/node_modules/cwebp-bin
> node lib/install.js

  ⚠ spawn /builds/backend/felix/front/node_modules/cwebp-bin/vendor/cwebp ENOENT
  ⚠ cwebp pre-build test failed
  ℹ compiling from source
  ✖ Error: Command failed: /bin/sh -c ./configure --disable-shared --prefix="/builds/backend/felix/front/node_modules/cwebp-bin/vendor" --bindir="/builds/backend/felix/front/node_modules/cwebp-bin/vendor"
configure: error: in `/builds/backend/felix/front/node_modules/cwebp-bin/e57e42a0-d0ae-4cbe-8a10-4ab0dc44209d':
configure: error: no acceptable C compiler found in $PATH
See `config.log' for more details

checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... ./install-sh -c -d
checking for gawk... no
checking for mawk... no
checking for nawk... no
checking for awk... awk
checking whether make sets $(MAKE)... no
checking whether make supports nested variables... no
checking for style of include used by make... none
checking for gcc... no
checking for cc... no
checking for cl.exe... no

    at Promise.all.then.arr (/builds/backend/felix/front/node_modules/bin-build/node_modules/execa/index.js:231:11)
    at processTicksAndRejections (internal/process/task_queues.js:86:5)

> gifsicle@4.0.1 postinstall /builds/backend/felix/front/node_modules/gifsicle
> node lib/install.js

  ⚠ spawn /builds/backend/felix/front/node_modules/gifsicle/vendor/gifsicle ENOENT
  ⚠ gifsicle pre-build test failed
  ℹ compiling from source
  ✖ Error: Command failed: /bin/sh -c autoreconf -ivf
/bin/sh: autoreconf: not found


    at Promise.all.then.arr (/builds/backend/felix/front/node_modules/bin-build/node_modules/execa/index.js:231:11)
    at processTicksAndRejections (internal/process/task_queues.js:86:5)
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.7 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! gifsicle@4.0.1 postinstall: `node lib/install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the gifsicle@4.0.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-05-31T20_04_56_307Z-debug.log
ERROR: Job failed: exit code 1
```

</details>

--- a/package.json
+++ b/package.json
@@ -18,14 +18,16 @@
   },
   "dependencies": {
     "imagemin": "^5.3.1",
+    "loader-utils": "^1.1.0",
+    "object-assign": "^4.1.1"
+  },
+  "optionalDependencies": {
     "imagemin-gifsicle": "^6.0.0",
     "imagemin-mozjpeg": "^8.0.0",
     "imagemin-optipng": "^6.0.0",
     "imagemin-pngquant": "^6.0.0",
     "imagemin-svgo": "^7.0.0",
-    "imagemin-webp": "^5.0.0",
-    "loader-utils": "^1.1.0",
-    "object-assign": "^4.1.1"
+    "imagemin-webp": "^5.0.0"
   },
   "devDependencies": {
     "file-loader": "^1.1.11",

